### PR TITLE
Change code language blm

### PR DESCRIPTION
**Pull request by Fasse**
#937

### CODE REVIEW

File: adm_program/languages/de.xml [hunk 1]
```diff
@@ -741,6 +741,7 @@
   <string name="SYS_BACKUP_FILE">Backupdatei</string>
   <string name="SYS_BACKUP_FROM">Backup vom #VAR1# um #VAR2#</string>
   <string name="SYS_BACKUP_ONLY_MYSQL">Das Backupmodul kann nur bei einer MySQL-Datenbank genutzt werden.</string>
+  <string name="SYS_BASIC_DATA">Basic data</string>
```
Comment (ximex) [inline adm_program/languages/de.xml:744]: should be german

**Review by ximex — CHANGES_REQUESTED** [review]
@Fasse i'm not sure about `whitespace` and `empty space`. I never heard about `empty space`. And in this case the "white" from `whitespace` depends on the white paper and the missing black ink. But not sure

File: adm_program/languages/de.xml (hunk 1 above)
Comment (Fasse) [reply adm_program/languages/de.xml:744]: What do you think is a good name **Basisdaten** or **Grundlegende Daten**?

Comment (ximex) [reply adm_program/languages/de.xml:744]: `Basisdaten`

Comment (Fasse) [reply adm_program/languages/de.xml]: > @Fasse i'm not sure about `whitespace` and `empty space`. I never heard about `empty space`. And in this case the "white" from `whitespace` depends on the white paper and the missing black ink. But not sure

Ok, if you like we could stay with whitespace. Give me a short feedback.

**Pull request merged**